hive-hooks/path-conversion-hook/src/main/java/com/expediagroup/apiary/extensions/hooks/pathconversion/config/Configuration.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (C) 2018-2021 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.apiary.extensions.hooks.pathconversion.config;
+
+import static java.lang.String.format;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Properties;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang.BooleanUtils;
+import org.apache.hadoop.hive.conf.HiveConf;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+import com.google.common.collect.ImmutableList;
+
+import com.expediagroup.apiary.extensions.hooks.pathconversion.models.PathConversion;
+
+@Slf4j
+@Getter
+public class Configuration {
```

Comment:
Based on the above discussion, I guess this would need to go in the `pathconversion` package with a very simple/marker interface/base class for the more generic stuff?